### ISSUE TITLE
[Feature Request] Typing UCSUR Unicode codepoints

### ISSUE THREAD

**Issue by Susul-1312** [timeopochin/tokiponakeyboard#26]
While Toki Pona is not in Unicode, it is in the [Under-ConScript Unicode Registry](https://www.kreativekorp.com/ucsur/), which assigns "private use" codepoints from unicode to various conscripts. It would be awesome if you could type [the codepoints for toki pona as designated by them](https://www.kreativekorp.com/ucsur/charts/sitelen.html) using this keyboard, maybe in a different mode from the latin script one. using these codepoints has the advantage that you can have latin text next to toki pona text without much effort, as long as your font supports it. An example where its used is a discord modification patching in such a font, so you can read messages in sitelen pona without it messing with normal latin messages.

TL;DR: make [these codepoints typable pretty please](https://www.kreativekorp.com/ucsur/charts/sitelen.html)

PS: Love this app
